egs/wsj/s5/steps/online/nnet3/decode.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
+
+# Copyright 2014  Johns Hopkins University (Author: Daniel Povey)
+#           2016  Api.ai (Author: Ilya Platonov)
+# Apache 2.0
+
+# Begin configuration section.  
+stage=0
+nj=4
+cmd=run.pl
+max_active=7000
+threaded=false
```

Comment:
Done!

On Thu, Apr 14, 2016 at 5:53 PM, Ilya Platonov <notifications@github.com>
wrote:

> In egs/wsj/s5/steps/online/nnet3/decode.sh
> <https://github.com/kaldi-asr/kaldi/pull/666#discussion_r59798533>:
>
> > @@ -0,0 +1,154 @@
> > +#!/bin/bash
> > +
> > +# Copyright 2014  Johns Hopkins University (Author: Daniel Povey)
> > +#           2016  Api.ai (Author: Ilya Platonov)
> > +# Apache 2.0
> > +
> > +# Begin configuration section.
> > +stage=0
> > +nj=4
> > +cmd=run.pl
> > +max_active=7000
> > +threaded=false
>
> Done, I assume egs/swbd/s5c/local/chain/run_tdnn_6v.sh should be done by
> @pegahgh <https://github.com/pegahgh> , since I am bit out of context
> here.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/666/files/7dda2e6c24677d66dc270da9a9750fd77453aeb8#r59798533>
>